packages/gatsby-theme-store/src/gatsby-node.ts

```diff
@@ -206,6 +206,15 @@ export const onCreateWebpackConfig = (
 
   setWebpackConfig({
     ...profilingConfig,
+    resolve: {
+      alias: {
+        ...profilingConfig?.resolve.alias,
+        '@vtex/order-manager': require.resolve(
```

Comment:
@tlgimenes I did this, to not use a compiled files from this both packages. Because of this change, LH passes on btglobal.store (PR link on description)

Reply:
No, you should do this on `@vtex/order-manager`'s package.json file instead to avoid creating long webpack configs. Like so, we may also benefit from this on IO